SECURITY.md

```diff
@@ -0,0 +1,29 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+For *critical* bugs, please send an email to security@filecoin.org.
```

Comment:
Wanna set that in [email syntax](https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses), or are you afraid of spam?

Reply:
i'm afraid of the email syntax

Reply:
when rendered it already links to the email (i guess it's some github magic)